package.nls.json

```diff
@@ -37,6 +37,7 @@
     "AWS.configuration.description.samcli.location": "Location of SAM CLI. SAM CLI is used to create, build, package, and deploy Serverless Applications. [Learn More](https://aws.amazon.com/serverless/sam/)",
     "AWS.configuration.description.telemetry": "Enable AWS Toolkit to send usage data to AWS.",
     "AWS.configuration.description.telemetry.cn": "Enable Amazon Toolkit to send usage data to Amazon.",
+    "AWS.configuration.description.doNotShowPrompts": "List of prompts not to show again.",
```

Comment:
wording suggestion: "List of prompts to suppress"

Reply:
Thanks for the suggestion. Definitely an improvement.  Change added.